Apps/W1/HybridGP/app/src/Migration/History/InstallGPSmartLists.codeunit.al

```diff
@@ -1,10 +1,14 @@
 codeunit 4033 "Install GP SmartLists"
 {
+    ObsoleteState = Pending;
```

Comment:
You should add pragma statements around
e.g. #if not CLEAN22
#endif

Then we can auto remove the code